plugins/modules/monitoring/uptimerobot.py

```diff
@@ -8,138 +8,181 @@
 
 
 DOCUMENTATION = '''
-
 module: uptimerobot
-short_description: Pause and start Uptime Robot monitoring
+short_description: Create, remove, pause and start monitors in Uptime Robot
 description:
-    - This module will let you start and pause Uptime Robot Monitoring
-author: "Nate Kingsley (@nate-kingsley)"
+    - This module lets you create, remove, pause and start monitors in Uptime Robot.
+author:
+- "Nate Kingsley (@nate-kingsley)"
+- "Ninjiner (@Ninjiner)"
 requirements:
     - Valid Uptime Robot API Key
 options:
     state:
         description:
             - Define whether or not the monitor should be running or paused.
+            - C(created) and C(absent) were added in community.general 1.3.0.
```

Comment:
`present` wasn't there before, either.

Reply:
Changed in: https://github.com/ansible-collections/community.general/pull/961/commits/2b5ae1786b9cd7f4587b452ec7f38452a63d3394